<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ArtStyle</class>
 <widget class="QWizard" name="ArtStyle">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>628</width>
    <height>505</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="font">
   <font>
    <family>Century Gothic</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>ArtStyle</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>img/transfer_style_reference.jpg</normaloff>img/transfer_style_reference.jpg</iconset>
  </property>
  <property name="whatsThis">
   <string/>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <property name="wizardStyle">
   <enum>QWizard::AeroStyle</enum>
  </property>
  <property name="titleFormat">
   <enum>Qt::AutoText</enum>
  </property>
  <property name="subTitleFormat">
   <enum>Qt::AutoText</enum>
  </property>
  <widget class="QWizardPage" name="wizardPage1">
   <widget class="QLabel" name="ground_label">
    <property name="geometry">
     <rect>
      <x>-30</x>
      <y>-20</y>
      <width>771</width>
      <height>381</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>img/fon.jpg</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="text_label">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>0</y>
      <width>161</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>High Tower Text</family>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;</string>
    </property>
    <property name="text">
     <string>ArtStyle</string>
    </property>
   </widget>
   <widget class="QLabel" name="text_label_2">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>110</y>
      <width>301</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>High Tower Text</family>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;</string>
    </property>
    <property name="text">
     <string>Create art from fhoto</string>
    </property>
   </widget>
  </widget>
  <widget class="QWizardPage" name="wizardPage2">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>521</width>
      <height>57</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Загрузите изображение:</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLineEdit" name="lineEdit">
         <property name="text">
          <string>С:/</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>Обзор...</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="autoDefault">
          <bool>true</bool>
         </property>
         <property name="default">
          <bool>false</bool>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>120</y>
      <width>300</width>
      <height>200</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: lightgrey;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap/>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
    <property name="indent">
     <number>-1</number>
    </property>
   </widget>
  </widget>
  <widget class="QWizardPage" name="wizardPage3">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>541</width>
      <height>301</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::DefaultContextMenu</enum>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tabWidgetPage1">
     <attribute name="title">
      <string/>
     </attribute>
     <widget class="QLabel" name="label_style1">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>601</width>
        <height>431</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>img/s1.jpg</pixmap>
      </property>
      <property name="openExternalLinks">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabWidgetPage2">
     <attribute name="title">
      <string/>
     </attribute>
     <widget class="QLabel" name="label_style2">
      <property name="geometry">
       <rect>
        <x>-30</x>
        <y>0</y>
        <width>581</width>
        <height>331</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>img/s2.jpg</pixmap>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabWidgetPage3">
     <attribute name="title">
      <string/>
     </attribute>
     <widget class="QLabel" name="label_style3">
      <property name="geometry">
       <rect>
        <x>-10</x>
        <y>0</y>
        <width>561</width>
        <height>431</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>img/s3.jpg</pixmap>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabWidgetPage4">
     <attribute name="title">
      <string/>
     </attribute>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>201</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Загрузить новый стиль:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>50</y>
        <width>300</width>
        <height>200</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>img/w5floads.png</pixmap>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
      <property name="openExternalLinks">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>519</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Выберите стиль:</string>
    </property>
   </widget>
  </widget>
  <widget class="QWizardPage" name="wizardPage4">
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>80</y>
      <width>251</width>
      <height>23</height>
     </rect>
    </property>
    <property name="maximum">
     <number>2</number>
    </property>
    <property name="value">
     <number>-1</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>519</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Итоговый результат:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>170</y>
      <width>300</width>
      <height>200</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: lightgrey;</string>
    </property>
    <property name="text">
     <string>Поле для итогового фото</string>
    </property>
   </widget>
   <widget class="QPushButton" name="startTf">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>130</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Начать обработку</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
